src/api/userInformation.ts

```diff
@@ -0,0 +1,30 @@
+import axios from 'axios';
+import { url, headers } from '../shared/variable.shared';
+
+
+
+function getInformation(user: any){
+    return {
+        email: user.email,
+        password: "",
+        provider:"google"
+        // name: user.name ?? "",
+        // picture: user.picture ?? ""
+    }
+}
+
+
+async function sendUserInformation(user: any){
+    const information = getInformation(user);
+    let result = ''
+    if (url) 
+        await axios.post(url+'/users', information, { headers })
```

Comment:
Jak używasz awaita to zrób try catcha zamiast chainingu

Reply:
dlaczego to zamknąłeś bez żadnej odpowiedzi ani poprawki?

Reply:
zrobione